portal/models/i18n.py

```diff
@@ -0,0 +1,90 @@
+"""Module for i18n methods and functionality"""
+import psycopg2
+import sys
+import os
+from flask import current_app
+from ..extensions import babel
+from .user import current_user
+
+
+def parse_connection_uri():
+    here, _ = os.path.split(__file__)
+    with open(os.path.join(current_app.root_path, '../instance/application.cfg'), 'r') as fh:
+        conn_strings = [l for l in fh.readlines() if
+                        l.startswith('SQLALCHEMY_DATABASE_URI')]
+
+    if len(conn_strings) != 1:
+        raise ValueError("can't find connection string in application.cfg")
+    conn_uri = conn_strings[0].split('=')[1]
+    return conn_uri.strip()[1:-1]  # strip quotes, newlines
+
+
+def connect_to_db():
+    try:
+        connection_uri = parse_connection_uri()
+        conn = psycopg2.connect(connection_uri)
+        return conn.cursor()
+    except:
+        exceptionType, exceptionValue, exceptionTraceback = sys.exc_info()
+        sys.exit("Database connection failed!\n ->%s" % (exceptionValue))
+
+
+def add_to_dict(d,k,o):
+    if k:
+        key = "\"" + k + "\""
+        if key not in d:
+            d[key] = set()
+        d[key].add(o)
+
+
+def get_db_strings():
+    cur = connect_to_db()
+    if cur:
+        elements = {}
+        cur.execute("""SELECT custom_text, name FROM apptext""")
```

Comment:
As you've demonstrated, you can run raw SQL statements through the SQLAlchemy engine, but that should only be necessary in the exceptional case.

Catch me online for more help if you'd like.  Query each object for what you need directly, i.e.
```
query = AppText.query.with_entities(AppText.custom_text)
for entry in query:
```